### ISSUE TITLE
Open attachments in new windows/tab

### ISSUE THREAD

**Issue by OleSta** [osTicket/osTicket-1.7#758]
Why not open attachments in new windows/tab?

It could be a configuration option.

For now I just changed line 470 in class.thread.php
from:
function getAttachmentsLinks($file='attachment.php', $target='', $separator=' ') {
to:
function getAttachmentsLinks($file='attachment.php', $target='_blank', $separator=' ') {

**Comment by OleSta** [comment]
How do I do the same when printing a ticket - open in new windows/tab?

**Comment by greezybacon** [comment]
@OleSta, I agree the same tab is confusing. It just recently changed to inline download. One easy fix (documentation wise) is to CTRL+click to open in a new tab.

The hold up is to make sure if you were download a bunch of files a browser cannot render (like EXEs or ZIPs), if the browser would end up with lots of empty tabs.
